### PR TITLE
Update issue templates (remove issue report for bug report)

### PR DESCRIPTION
**Pull request by Rycia**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 1]
```diff
@@ -1,16 +1,18 @@
 ---
 name: Feature request
-about: Suggest an idea for Azeroth Pilot Reloaded
+about: Suggest an idea for Azeroth Pilot Reloaded.
 title: "[FEATURE] - "
-labels: "Feature"
+labels: Feature
```
Comment (Neogeekmo) [inline .github/ISSUE_TEMPLATE/feature-request.md:5]: why did you delete the "" they are not mandatory?

Comment (Rycia) [reply .github/ISSUE_TEMPLATE/feature-request.md:5]: i never did lol. github did it itself

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 2]
```diff
@@ -1,16 +1,18 @@
...
 assignees: ''
 
 ---
 
-### Please describe the issues and/or needs to which your feature request relates
+### Please describe the issues and/or needs to which your feature request relates to.
```
Comment (Pahonix) [inline .github/ISSUE_TEMPLATE/feature-request.md:10]: This change makes the sentence grammatically incorrect. It was correct in the previous iteration. One should not end a sentence with a preposition.

Comment (Rycia) [reply .github/ISSUE_TEMPLATE/feature-request.md:10]: alright

File: .github/ISSUE_TEMPLATE/feature-request.md (hunk 1 above)
Comment (Neogeekmo) [reply .github/ISSUE_TEMPLATE/feature-request.md:5]: so it's not useful when there is only one I think

**Review by Pahonix — CHANGES_REQUESTED** [review]
Revert line 10 change

File: .github/ISSUE_TEMPLATE/feature-request.md (hunk 2 above)
Comment (Neogeekmo) [inline .github/ISSUE_TEMPLATE/feature-request.md:10]:
```suggestion
### Please describe the issues and/or needs to which your feature request relates.
```

**Review by Neogeekmo — CHANGES_REQUESTED** [review]

**Review by Neogeekmo — APPROVED** [review]
you need to make a suggestion to apply a change ;)

**Review by Pahonix — APPROVED** [review]
